CHANGELOG/CHANGELOG-1.8.md

```diff
@@ -0,0 +1,99 @@
+- Add columns of kubectl outputs for Multi-cluster types. ([#3923](https://github.com/antrea-io/antrea/pull/3923), [@jianjuns])
+- Add IP version and prefix length validation to IPPool CRD manifest. ([#3897](https://github.com/antrea-io/antrea/pull/3897), [@jianjuns])
+- Add an output formatter "raw" to better display multi-line string responses for antctl. ([#3589](https://github.com/antrea-io/antrea/pull/3589), [@Atish-iaf])
+- Add new ports to network requirement doc. ([#4063](https://github.com/antrea-io/antrea/pull/4063), [@luolanzone])
+- Windows OVS installation script now installs required SSL library if missing. ([#4029](https://github.com/antrea-io/antrea/pull/4029), [@XinShuYang])
+- Upgrade whereabouts CNI to v0.5.4 and align secondary network IPAM with additional pluginArgs. ([#3987](https://github.com/antrea-io/antrea/pull/3987), [@arunvelayutham])
+- Remove Theia related files. ([#4048](https://github.com/antrea-io/antrea/pull/4048), [@dreamtalen])
+<!-- 
+- Use hostNetwork for Multi-cluster controller. ([#3965](https://github.com/antrea-io/antrea/pull/3965), [@luolanzone])
+- Update ClusterClaim to v1alpha2. ([#3755](https://github.com/antrea-io/antrea/pull/3755), [@bangqipropel])
+- Update GatewayIPPrecedence to support "external/internal" option for the Multi-cluster feature. ([#3930](https://github.com/antrea-io/antrea/pull/3930), [@luolanzone])
+- Disable metrics API and change the health binding address port to 8080 for the Multi-cluster feature. ([#4101](https://github.com/antrea-io/antrea/pull/4101), [@luolanzone])
+- Improve CRD validation for the Multi-cluster feature. ([#4062](https://github.com/antrea-io/antrea/pull/4062) [#4090](https://github.com/antrea-io/antrea/pull/4090), [@luolanzone])
+- Cleanup ResourceExport if exported Service has no available Endpoints. ([#4056](https://github.com/antrea-io/antrea/pull/4056), [@luolanzone])
+- Add controller for leader cluster to garbage collect MemberClusterAnnounce for the Multi-cluster feature. ([#4054](https://github.com/antrea-io/antrea/pull/4054), [@hjiajing])
+- Auto update ClusterSet and MemberClusterAnnounce in leader cluster. ([#3956](https://github.com/antrea-io/antrea/pull/3956), [@hjiajing])
+- Update ClusterSet and ClusterClaim webhooks. ([#4043](https://github.com/antrea-io/antrea/pull/4043), [@luolanzone])
+- Remove MemberClusterAnnounce if ClusterSet deleted. ([#4026](https://github.com/antrea-io/antrea/pull/4026), [@luolanzone])
+- Refactor Common Area and remove leader election. ([#3964](https://github.com/antrea-io/antrea/pull/3964), [@luolanzone])
+- Add more MC Service info and ToC to Multi-cluster user guide. ([#3909](https://github.com/antrea-io/antrea/pull/3909), [@jianjuns])
+- Revise Multi-cluster Gateway descriptions in user docs. ([#3899](https://github.com/antrea-io/antrea/pull/3899), [@jianjuns])
+- Update Multi-cluster architecture doc. ([#3638](https://github.com/antrea-io/antrea/pull/3638), [@luolanzone])
+-->
```

Comment:
@jianjuns @luolanzone there are many changes made to multi-cluster in this release and I'm not sure if all of them are important to be listed in release note or some of them are refining each other. Could you help point the ones worth to be mentioned?